### ISSUE TITLE
Week number - wrong language

### ISSUE THREAD

**Issue by Jannomag** [totaldebug/atomic-calendar-revive#774]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Week number is shown as "Week 34" (for example)

### Expected behaviour

Localized "Week"-String.

I already tried to replace the string "Week" in the javascript file but this doesn't work.

### Card Version

7.0.1

### Home Assistant Version

2022.8.6

### Configuration

```shell
type: custom:atomic-calendar-revive
entities:
  - entity: calendar.familie
    name: Familie
    color: '#FFB900'
    icon: mdi:human-male-female-child
  - entity: calendar.kita
    name: Kita
    color: '#00CC6A'
    icon: mdi:teddy-bear
  - entity: calendar.feiertage_in_deutschland
    name: Feiertage
    color: '#6B69D6'
    icon: mdi:beach
showDate: true
untilText: Bis
fullDayEventText: Ganztägig
finidshedEventOpcaity: 0.6
finishedEventFilter: grayscale(100%)
dimFinishedEvents: true
defaultMode: Event
showWeekDay: false
showLoader: true
showLastCalendarWeek: true
showRelativeTime: true
enableModeChange: true
calShowDescription: true
disableCalEventLink: false
disableCalLocationLink: false
disableCalLink: false
firstDayOfWeek: 1
refreshInterval: 60
dateFormat: dddd, DD. MMMM YYYY
timeColor: white
timeSize: 100
dateSize: 200
eventTitleSize: 120
showNoEventsForToday: true
style: |
  .week-number {
    color: rgba(255,255,255,0.6) !important;
    border-radius: 6px !important;
    border: none !important;
    margin 10px 0px !important;
    line-height: 24px !important;
    padding: 0px 0px 0px 0px !important;
  }
  ha-card {
    backdrop-filter: blur(50px) saturate(150%) brightness(50%);
    -webkit-backdrop-filter: blur(50px) saturate(150%) brightness(50%);
    border-radius: 10px;
    border: 1px solid rgba(255,255,255,0.05);
  }
  .header {
    height: 12px;
  }
showMonth: false
showProgressBar: false
showEventIcon: true
showLocation: false
showCalendarName: true
showHiddenText: false
showWeekNumber: true
showFullDayProgress: false
showCurrentEventLine: false
noEventsForNextDaysText: Keine Termine in nächster Zeit
noEventText: Keine Termine heute
showDeclined: false
maxDaysToShow: 7
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by marksie1988** [comment]
Please could you tell me what language you are trying to show the week in? (country code is best)

**Comment by Jannomag** [comment]
It's German (de / de-de)

**Comment by FloffyTheGreat** [comment]
Have the same experience with Danish (da-DK), lcid 1030

**Comment by marksie1988** [comment]
Hey, just tested with `de` in my user and this works: 

![image](https://user-images.githubusercontent.com/159166/209821374-40c2950f-93c9-428a-971e-5a95bdf26b61.png)
![image](https://user-images.githubusercontent.com/159166/209821461-88266755-025e-44b0-9efc-41e8a7924c41.png)

**Comment by marksie1988** [comment]
fixed in #720
